app/models/katello/subscription_status.rb

```diff
@@ -47,6 +48,8 @@ def to_status(options = {})
         PARTIAL
       when Katello::Candlepin::Consumer::ENTITLEMENTS_INVALID
         INVALID
+      when Katello::Candlepin::Consumer::UNSUBSCRIBED_HYPERVISOR
```

Comment:
Instead of this, we really should check for 'if host.subscription_facet.hypervisor? &&  host.subscription_facet.candlepin_consumer.entitlements.empty?'  here